test/spec/ModelerSpec.js

```diff
@@ -46,54 +47,129 @@ describe('modeler extension', function() {
 
     beforeEach(bootstrapModeler(diagram, {
       additionalModules: [
-        ...Modeler.prototype._modules,
         TokenSimulationModelerModules
       ]
     }));
 
-
     function expectColors(elementId, expectedColors) {
-
       return getBpmnJS().invoke(function(elementRegistry, elementColors) {
         const element = elementRegistry.get(elementId);
 
         expect(element).to.exist;
 
-        const colors = elementColors.get(element);
+        const colors = elementColors._get(element);
```

Comment:
Why is this a private API now?

Reply:
Because it's not used by anyone else anymore. `add` and `remove` should be sufficient as public API.

Reply:
:+1: 

Just feels odd we need to use internals inside our test.